sklearn/inspection/_partial_dependence.py

```diff
@@ -36,7 +36,7 @@
 ]
 
 
-def _grid_from_X(X, percentiles, grid_resolution):
+def _grid_from_X(X, percentiles, grid_resolution, custom_values):
```

Comment:
is it possible to default `custom_values` as an empty dict? This avoids the case where a value needs to be passed in to this as an argument when it isn't necessary

Reply:
I believe it needs to default to `None`, then be replaced by an empty dictionary if it's `None`.